### DISCUSSION TITLE
where to find the setTransactionId

### DISCUSSION THREAD

**Discussion by mohd786hussain** [PatilShreyas/EasyUpiPayment-Android#94] [Q&A]
Can you post the complete steps for this as how and where to find the setTransactionId ??? Much appreciated in advance.

**Comment by PatilShreyas** [comment]
You can refer to this documentation for complete reference: https://patilshreyas.github.io/EasyUpiPayment-Android/#/pages/setup-payment-info

**Reply by PatilShreyas** [reply]
Alternatively, you can refer to the demo source code: https://github.com/PatilShreyas/EasyUpiPayment-Android/blob/master/app/src/main/java/com/example/easyupipayment/java/MainActivity.java
